sdk/storage/azure-storage-blob-batch/src/main/java/com/azure/storage/blob/batch/BlobBatch.java

```diff
@@ -136,12 +139,15 @@
      * @throws UnsupportedOperationException If this batch has already added an operation of another type.
      */
     public Response<Void> deleteBlob(String containerName, String blobName) {
-        return deleteBlobHelper(String.format(PATH_TEMPLATE, containerName, blobName), null, null);
+        return deleteBlobHelper(String.format(PATH_TEMPLATE, containerName,
```

Comment:
This should just call into the next overload so we don't have to worry about missing formatting/encoding logic in one of them

Reply:
You are right, let's make it call the next overload.